BookingsApi/BookingsApi.IntegrationTests/Services/HearingAllocationService/CheckForAllocationClashesTests.cs

```diff
@@ -52,10 +53,40 @@ public void should_set_work_hour_clashes_to_true_when_hearing_exceeds_cso_workin
             .Build();
         hearing.Allocations.Add(new Allocation {Hearing = hearing, JusticeUser = cso});
 
+        // act
         var resultDtos = _sut.CheckForAllocationClashes(new List<VideoHearing> {hearing});
+        
+        // assert
         resultDtos.Count.Should().Be(1);
         resultDtos[0].HasWorkHoursClash.Should().BeTrue();
     }
+    
+    [Test]
+    public void should_set_exceeded_concurrency_to_true_when_more_than_three_hearings_overlap()
```

Comment:
Update the test name from true to counter

Reply:
Done